.github/scripts/js/e2e/slash_workflow_command.js

```diff
@@ -0,0 +1,93 @@
+const {abortFailedE2eCommand} = require("../constants");
+
+/**
+ * Build valid return object
+ * @param {string} flag for realease issue logic
+ * @param {object} workflow_id - workflow id
+ * @param {object} targetRef - workflow target ref
+ * @param {object} inputs - target workflow inputs
+ * @return {object}
+ */
+function buildReturn(flag, workflow_id, targetRef, inputs) {
+  return {
+    [flag]: true,
+    workflow_id,
+    targetRef,
+    inputs,
+  }
+}
+
+/**
+ * Try parse e2e abort arguments
+ * @param {object} inputs
+ * @param {object} inputs.core - A reference to the '@actions/core' package.
+ * @param {object} inputs.context - A reference to context https://github.com/actions/toolkit/blob/main/packages/github/src/context.ts#L6
+ * @param {string[]} inputs.argv - array of slash command argv[0] is commnad
+ * @return {object}
+ */
+function tryParseAbortE2eCluster({argv, context, core}){
+  const command = argv[0];
+  if (command !== abortFailedE2eCommand) {
+    return null;
+  }
+
+  // example
+  // /e2e/abort static;Static;containerd;1.21 3318607912 3318607912-1-con-1-21
+  // explain:
+  // /e2e/abort - command
+  // static;Static;containerd;1.21 - run parameters (provider;layout;cri;k8s version)
+  // 3318607912 - run id (needs for get artifact)
+  // 3318607912-1-con-1-21 - cluster prefix (needs for run dhctl bootstrap-phase abort command)
+  // /sys/deckhouse-oss/install:pr2896 - install image path: for run necessary installer
+  // user@127.0.0.1 - [additional] connection string, needs for fully bootstrapped cluster, but e2e was failed.
+  //                  we  need it for destroy
+  if (argv.length < 5) {
+    return {err: 'clean failed e2e cluster should have 4 arguments'};
+  }
```

Comment:
Something wrong with these conditions:
- argv.length < 5
- should have 4 arguments

Reply:
it's not mistake. `argv[0]` is command, `argv[1..4]` is arguments.